code/world.dm

```diff
@@ -17,6 +17,7 @@
 	hub_password = "kMZy3U5jJHSiBQjr"
 	name = "/tg/ Station 13"
 	fps = 20
+	cache_lifespan = 0 // SKYRAT EDIT - Makes sure TTS shit isnt kept past rounds
```

Comment:
Is this sufficient for nonmodular markage.

Reply:
yep, for simple things like this it's generally preferred to be a modular override of `/world` in the `modular_skyrat` folder, but it's not a massive deal for a one-liner like this